### PR TITLE
Drop transpilation and stop polluting global

### PR DESCRIPTION
**Pull request by paed01**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -82,18 +81,24 @@ function Tallahassee(app, options = {}) {
...
     let elementsToScroll = () => {};
     const stickedElements = [];
 
-    compile();
-
     const headers = getHeaders(resp.request);
+    const document = Document(resp, agent.jar);
     const window = Window(resp, {
       fetch: Fetch(agent, resp),
+      get document() {
```
Comment (andolf) [inline index.js]: What is this I don't even

File: package.json [hunk 2]
```diff
@@ -1,23 +1,24 @@
 {
   "name": "@expressen/tallahassee",
-  "version": "8.1.0",
+  "version": "9.1.0",
```
Comment (andolf) [inline package.json]: Should this be version 9.0.0?

Comment (paed01) [reply package.json]: Yep, breaking. But to keep up with master it is now on 9.1

File: index.js (hunk 1 above)
Comment (paed01) [reply index.js]: instead of `Object.defineProperty(window, "document", { get() { return document; } }`. ES6.

File: package.json (hunk 2 above)
Comment (andolf) [reply package.json]: README says `9.0.0` however. Still correct?

**Comment by andolf** [comment]
One conflict. Otherwise looks good :+1:

**Comment by andolf** [comment]
I'll merge this in the week. We're still tinkering with some other stuff that we probably don't want to update to a major for.

**Comment by CaptainJiNX** [comment]
What are the thoughts about removing setTimeout, and how is that supposed to be tested? (This version breaks our tests since we are using window.setTimeout...)

**Comment by paed01** [comment]
@CaptainJiNX Well, the idea is that you should control your own globals. That goes for timers as well.
```js
const browser = await Tallahassee(app).navigateTo("/");
browser.window.setTimeout = () => {};
```

**Pull request merged**